docs/endpoints.rst

```diff
@@ -68,7 +65,29 @@ For example
     'endpoint_address': address_by_hostname(),
   }
 
-.. include:: configuring.rst
+
+FuncX endpoints act as gateways to diverse computational resources, including clusters, clouds,
+supercomputers, and even your laptop. To make the best use of your resources, the endpoint must be
+configured to match the capabilities of the resource on which it is deployed.
+
+FuncX provides a Python class-based configuration model that allows you to specify the shape of the
+resources (number of nodes, number of cores per worker, walltime, etc.) as well as allowing you to place
+limits on how funcX may scale the resources in response to changing workload demands.
+
+To generate the appropriate directories and default configuration template, run the following command::
+
+  $ funcx-endpoint configure <ENDPOINT_NAME>
+
+This command will create a profile for your endpoint in `$HOME/.funcx/<ENDPOINT_NAME>/` and will instantiate a
```

Comment:
In a different part of this PR, Yadu suggested that this file is no longer used.